straxen/contexts.py

```diff
@@ -2,6 +2,14 @@
 import strax
 import straxen
 
+# list of contexts that gets tracked in runDB context collection
+# needs to be maintained for each straxen release
+context_list = ['xenonnt_led', 
```

Comment:
Would it not make sense to put this somewhere else? If a user would do e.g.  straxen.contexts.context_list, I would be expecting it would list all the available contexts or so, which this isn't.

Reply:
moving it to the github/scripts would make much more sense I think

Reply:
Sure we can move it to .github/scripts, just need to have ACs (or whoever) remember that's where the list is maintained. I thought it might be useful for the list to be importable, but that's fine.